p11/index.html

```diff
@@ -1 +1,40 @@
-<!-- TODO -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h2>KWEB Survey Page</h2>
+    <img src="images/logo.png" width="250" alt="kweb logo"><br>
+    <a href="https://kwebofficial.com/" title="kweb homepage" target="_blank">KWEB Homepage</a>
+    <div class="question" id="question-1">
+        <h4>1. Write your name.</h4>
+        <input type="text" name="" id="user-name">
+    </div>
+    <div class="question" id="question-2">
+        <h4>2. Write tour student number.</h4>
+        <input type="number" name="" id="student-number">
+    </div>
+    <div class="question" id="question-3">
+        <h4>3.vSelect your member level.</h4>
+        <label><input type="radio" name="member-level" value="admin">Admin</label>
+        <label><input type="radio" name="member-level" value="intern">Intern</label>
+        <label><input type="radio" name="member-level" value="qualified">Qualified</label>
+    </div>
+    <div class="question" id="question-4">
+        <h4>4. Select all fields you are interested in.</h4>
+        <label><input type="checkbox" name="interest" value="front-end">Front-end</label>
+        <label><input type="checkbox" name="interest" value="back-end">Back-end</label>
+        <label><input type="checkbox" name="interest" value="web-crawling">Web crawling</label>
+        <label><input type="checkbox" name="interest" value="web-security">Web security</label>
+        <label><input type="checkbox" name="interest" value="etcetera">Etcetera</label> <input type="text" name="interest" value="etcetera-text">
```

Comment:
```suggestion
        <label><input type="checkbox" name="interest" value="etcetera">Etcetera</label> <input type="text" name="interest" placeholder="etcetera-text">
```
일반적으로는 placeholder를 많이 사용합니다!